src/main/java/uk/gov/hmcts/reform/finrem/notifications/controllers/NotificationController.java

```diff
@@ -128,4 +131,20 @@
         emailService.sendConfirmationEmail(notificationRequest, FR_CONTESTED_HWF_SUCCESSFUL);
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
+
+    @PostMapping(path = "/contested/prepare-for-hearing", consumes = APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "send e-mail for 'Prepare for hearing'.")
+    @ApiResponses(value = {
+        @ApiResponse(code = 204, message = "Prepare for hearing e-mail sent successfully")})
+    public ResponseEntity<Void> sendPrepareForHearingEmail(
+        @RequestBody
+        @ApiParam(value = "The fixtures contains case reference number, "
+            + "solicitorReferenceNumber and the email address that will receive "
+            + "the notification that the case is in the 'Prepare for a hearing' state")
+        @Valid final NotificationRequest notificationRequest) {
+        log.info("Received request to send email to Solicitor for 'Prepare for hearing' for Case ID: {}",
+            notificationRequest);
```

Comment:
No need to output the entire 'notificationRequest' object, should just output the case ID by changing this to: 'notificationRequest.getCaseReferenceNumber()'

Reply:
Good spot. Thanks for the suggestion. Updated now and pushed